test/mainline_test.go

```diff
@@ -125,6 +126,38 @@ var _ = Describe("Mainline component function tests", func() {
 
 		It("Should install resources for a CRD", func() {
 			stopChan := installResourceCRD(c, mgr)
+
+			instance := &operator.Installation{
+				TypeMeta:   metav1.TypeMeta{Kind: "Installation", APIVersion: "operator.tigera.io/v1"},
+				ObjectMeta: metav1.ObjectMeta{Name: "default"},
+			}
+			By("Checking that the installation status is set correctly")
+			Eventually(func() error {
+				err := GetResource(c, instance)
+				if err != nil {
+					return err
+				}
+				if instance.Status.Variant != operator.Calico {
+					return fmt.Errorf("installation status not Calico yet")
+				}
+				return nil
+			}, 60*time.Second).Should(BeNil())
+
+			By("Checking that the installation status does not change")
+			Consistently(func() error {
```

Comment:
Why will this not change? shouldn't the Variant get written?

Reply:
The block just above checks that the variant is written before continuing.

Reply:
Oh right, I was reading that previous one as checking that Variant was not Calico. My bad.